pycaption/base.py

```diff
@@ -217,6 +240,15 @@ def _format_timestamp(self, value, msec_separator=None):
         return u'0' + str_value
 
 
+class CaptionList(list):
+    """ A list of captions with a layout object attached to it """
+    def __init__(self, layout_info=None):
```

Comment:
Not sure what happens if the super method isn't called, but I'd call it.
Also, could you also leave in the first positional argument `iterable` so that this looks more like a list?